src/services/validate.ts

```diff
@@ -193,7 +192,7 @@ export default class ChartValidatorService extends BaseService {
   private async findLookups(): Promise<ServiceResponse<string>> {
     // Find any instance of "lookup" that starts with an opening parenthesis and ignore any white spaces between opening
     // and the word itself
-    const grepLookup = spawnSync('grep', ['-Rilne', '"(\s*lookup"', `"${this.toValidate}"`], {shell: true, encoding: "utf-8"});
+    const grepLookup = spawnSync('grep', ['-Rilne', '"\{\{\s*lookup"', `"${this.toValidate}"`], {shell: true, encoding: "utf-8"});
```

Comment:
Adding an optional `-` just in case is used as:
```yaml
{{- lookup...}}
```

```suggestion
    const grepLookup = spawnSync('grep', ['-Rilne', '"\{\{-?\s*lookup"', `"${this.toValidate}"`], {shell: true, encoding: "utf-8"});
```